inc/config.class.php

```diff
@@ -38,6 +38,20 @@
    die("Sorry. You can't access this file directly");
 }
 
+
+class ConfigPasswordTooWeak extends Exception {
```

Comment:
I'm not sure about the Exception class in config file.
Maybe we could add a new sub exceptions directory into inc to store this first one (and next).